src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-use-before-define */
 import * as React from 'react';
 import AsyncStorage from '@react-native-async-storage/async-storage';
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PropTypes` has been removed. If PropTypes are not being used in the component, this is a good practice to remove unused imports as it can improve the readability and maintainability of the code.
```diff
- import PropTypes from 'prop-types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->